src/Infrastructure/Web/Documentation/AuthResponsesOperationFilter.cs

```diff
@@ -1,19 +1,23 @@
 namespace Skeleton.Web.Documentation
 {
     using System.Linq;
-    using Microsoft.AspNetCore.Mvc.Authorization;
+    using Microsoft.AspNetCore.Authorization;
     using Swashbuckle.AspNetCore.Swagger;
     using Swashbuckle.AspNetCore.SwaggerGen;
 
     public class AuthResponsesOperationFilter : IOperationFilter
     {
         public void Apply(Operation operation, OperationFilterContext context)
         {
-            if (context.ApiDescription.ActionDescriptor.FilterDescriptors.Any(
-                    x => x.Filter.GetType().IsAssignableFrom(typeof(AuthorizeFilter))
-                )
-            )
-                operation.Responses.Add("401", new Response {Description = "Unauthorized"});
+            var authAttributes = context?.MethodInfo?.DeclaringType?.GetCustomAttributes(true)
+                .Union(context.MethodInfo.GetCustomAttributes(true))
```

Comment:
If you check `context` and `MethodInfo` for `null` in previous line, you will have to do it in current.